Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -425,9 +425,11 @@ public static Command ResetGlobalInputAsset(Action<InputActionAsset> postResetAc
         {
             return (in InputActionsEditorState state) =>
             {
+                // TODO Here we would instead modify the asset based on an instance loaded from defaults. Note that we cannot use importer here since we modify an in-memory asset as serialized object.... needs some thought to not break Undo.
```

Comment:
Its not immediately obvious in the UX that the reset doesn't work yet so need to make sure this is revisited.

That said the 'reset' button is currently missing from the UX.